rules/windows/lateral_movement_gpo_schtask_service_creation.toml

```
@@ -0,0 +1,55 @@
+[metadata]
+creation_date = "2020/08/13"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/08/13"
+
+[rule]
+author = ["Elastic"]
+description = """
+Detects creation of new Group Policy based Scheduled Task or Service. Both methods are used for legit system
+administration but can be abused by an attacker with domain admin level to execute a malicious payload remotely on all
+or a subset of the domain joined machines. Verify this change context such as username, process and GPO XML content itself.
```

```suggestion
Detects the creation or modification of a new Group Policy based scheduled task or service. These methods are used for legitimate system
administration, but can also be abused by an attacker with domain admin permissions to execute a malicious payload remotely on all
or a subset of the domain joined machines. 
```
Would move the below to `note`
> Verify the context changed, such as username, process name, and the GPO XML content itself.